schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
```

Comment:
- [OPTIONAL] What if we make the id column not null 🤔 
   eg:  `id int NOT NULL IDENTITY PRIMARY KEY`

Reply:
If you attempt to [insert](https://www.sqltutorial.org/sql-insert/) (or [update](https://www.sqltutorial.org/sql-update/)) a value like null into the GENERATED ALWAYS AS IDENTITY column, the database system will raise an error.

Reply:
Alright @rindrajosia, he can just skip that since it's an optional request!!